pkg/integrations/grok_exporter/config.go

```diff
@@ -0,0 +1,132 @@
+package grok_exporter //nolint:golint
+
+import (
+	"fmt"
+	v3 "github.com/fstab/grok_exporter/config/v3"
+	"github.com/fstab/grok_exporter/exporter"
+	"github.com/fstab/grok_exporter/oniguruma"
+	"github.com/go-kit/kit/log"
+	"github.com/grafana/agent/pkg/integrations"
+	"github.com/grafana/agent/pkg/integrations/config"
+	"gopkg.in/yaml.v3"
+)
+
+// Config controls the grok_exporter integration.
+type Config struct {
+	Common config.Common `yaml:",inline"`
+
+	GrokConfig v3.Config `yaml:",inline"`
+
+	IncludeExporterMetrics bool `yaml:"include_exporter_metrics"`
```

Comment:
Nit: can you remove the extra newlines here?

Reply:
Done